operations/hardware_state_manager/Lock_and_Unlock_Management_Nodes.md

```diff
@@ -44,7 +46,7 @@ ncn# kubectl -n services get pods | grep smd
 
 Example output:
 
-```
+```screen
```

Comment:
I recently learned that "screen" isn't one of the standard options for this. (When I looked to find the list of what is standard, a github.com document linked to this: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml)

```suggestion
```text
```